packages/autorest.typescript/src/restLevelClient/transforms/transformParameterTypes.ts

```diff
@@ -0,0 +1,208 @@
+import {
+  CodeModel, Operation, Parameter, ParameterLocation, Property, SchemaContext,
+  Request as M4OperationRequest
+} from "@autorest/codemodel";
+import { ImportKind, OperationParameter, ParameterBodyMetadata, ParameterBodySchema, ParameterMetadata, ParameterMetadatas, Schema } from "@azure-tools/rlc-codegen";
+import { getLanguageMetadata } from "../../utils/languageHelpers";
+import { NameType, normalizeName } from "../../utils/nameUtils";
+import { getDocs } from "../getPropertySignature";
+import { getOperationParameters } from "../helpers/operationHelpers";
+import { getElementType, primitiveSchemaToType } from "../schemaHelpers";
+
+export function transformParameterTypes(model: CodeModel, importDetails: Map<ImportKind, Set<string>>) {
+  const rlcParameters: OperationParameter[] = [];
+  let importedModels = new Set<string>();
```

Comment:
can we use Map<ImportKind, Set<string>> for this as well?

Reply:
Maybe I will change the name here 
- importedModels: Set 
- the inputted one - importDetails: Map<ImportKind, Set<string>>

So back to your question I think we use the Map one already importedModels is the internal Set